### PR TITLE
feat(black): add completion spec

### PR DESCRIPTION
**Pull request by msonnberger**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
feature

**What is the current behavior? (You can also link to an open issue here)**
No autocomplete for Black

**What is the new behavior (if this is a feature change)?**
Add autocomplete spec for [Black](https://github.com/psf/black) code formatter

### CODE REVIEW

File: src/black.ts [hunk 1]
```diff
@@ -0,0 +1,209 @@
+const blackVersions: Fig.Generator = {
```
Comment (fedeci) [inline src/black.ts:1]:
```suggestion
// https://github.com/psf/black
const blackVersions: Fig.Generator = {
```

File: src/black.ts [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    });
+  },
+};
+
+const completionSpec: Fig.Spec = {
+  name: "black",
+  description: "Python code formatter",
+  args: {
+    name: "file or directory",
+    template: ["filepaths", "folders"],
```
Comment (fedeci) [inline src/black.ts:19]: nit: folder is always added as part of `filepaths`.
```suggestion
    template: ["filepaths"],
```

File: src/black.ts [hunk 3]
```diff
@@ -0,0 +1,209 @@
+const blackVersions: Fig.Generator = {
+  script: "gh release list --repo psf/black",
```
Comment (fedeci) [inline src/black.ts:2]: We can add a couple days of cache here, WDYT?
```suggestion
  script: "gh release list --repo psf/black",
  cache: {
    ttl: 1000 * 60 * 60 * 24 * 2, // 2 days
  },
```

**Review by fedeci — CHANGES_REQUESTED** [review]
This looks really good, I left a couple comments to resolve.

File: src/black.ts (hunk 3 above)
Comment (msonnberger) [reply src/black.ts:3]: sure, that makes sense! I've added your suggestions.

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**